src/main/java/com/noroff/mefit/backup/BackUp.java

```diff
@@ -0,0 +1,83 @@
+package com.noroff.mefit.backup;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Calendar;
+import java.util.Map;
+
+public class BackUp {
+    public static void DumpBackup() {
+        Process p;
+        ProcessBuilder pb;
+        pb = new ProcessBuilder(
+                "C:\\Program Files\\PostgreSQL\\14\\bin\\pg_dump.exe",
```

Comment:
Is this a relative path that works in Heroku or only locally?

Reply:
This only works localy i am afraid